### PR TITLE
Deprecate `Request.failableSend()`

### PR DESCRIPTION
**Pull request by norio-nomura**
~~Use `Request.failableSend()` instead of `Request.send()`~~
Change `Request.send()` to `throws`

Some APIs changed to `throws`:
* `File.format(trimmingTrailingWhitespace:useTabs:indentWidth:) throws`
* `Structure.init(file:) throws`
* `SyntaxMap.init(file:) throws`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@
 * SourceKitten now requires Xcode 9 and Swift 3.2+ to build.  
   [Norio Nomura](https://github.com/norio-nomura)
 
+* Deprecated `Request.send()`.
```
Comment (jpsim) [inline CHANGELOG.md]: "Please use `Request.failableSend()` instead.

File: Source/SourceKittenFramework/Clang+SourceKitten.swift [hunk 2]
```diff
@@ -217,7 +222,9 @@ extension CXCursor {
             return xml.replacingOccurrences(of: " & ", with: " &amp; ")
         }
 
-        let cursorInfo = Request.cursorInfo(file: swiftUUID, offset: usrOffset, arguments: compilerArguments).send()
+        guard let cursorInfo = try? Request.cursorInfo(file: swiftUUID, offset: usrOffset, arguments: compilerArguments).failableSend() else {
+                return nil
```
Comment (jpsim) [inline Source/SourceKittenFramework/Clang+SourceKitten.swift]: indentation

**Comment by norio-nomura** [comment]
Thanks for reviews! Updated.  ✨

**Comment by norio-nomura** [comment]
@jpsim your CircleCI account seems to be stopped by following error:
> Your 14 day macOS trial has ended. Please go to the macOS plan page and select a plan to continue building.

https://circleci.com/gh/jpsim/SourceKitten/619

**Comment by jpsim** [comment]
CircleCI support granted the jpsim GitHub org with access to the Seed plan in early October, but I've exchanged over a dozen emails with them since that point because this has never been reflected in the CircleCI settings dashboard.

I just got an email from them a few minutes ago asking to check to see if it was working now, so I rebased your last commit and pushed again to see if it would run, and it did.

So we should be all good now.

Thanks for the PR!

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,15 @@
 * SourceKitten now requires Xcode 9 and Swift 3.2+ to build.  
   [Norio Nomura](https://github.com/norio-nomura)
 
+* Deprecated `Request.send()`. Please use `Request.failableSend()` instead.
```
Comment (jpsim) [inline CHANGELOG.md]: I feel like we should instead rename `failableSend()` to just `send()`. Throwing APIs in the Swift stdlib aren't prefixed with `failable`. It made sense to call it that when we had both, but now that we're removing that, I don't think it's appropriate anymore.

**Comment by norio-nomura** [comment]
Rebased. ✨

**Comment by norio-nomura** [comment]
Thanks! 🙏

**Pull request merged**